docs/data-sources/resources.md

```diff
@@ -0,0 +1,70 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_resources Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Resources in Twingate represent servers on the private network that clients can connect to. Resources can be defined by IP, CIDR range, FQDN, or DNS zone. For more information, see the Twingate documentation https://docs.twingate.com/docs/resources-and-access-nodes.
+---
+
+# twingate_resources (Data Source)
+
+Resources in Twingate represent servers on the private network that clients can connect to. Resources can be defined by IP, CIDR range, FQDN, or DNS zone. For more information, see the Twingate [documentation](https://docs.twingate.com/docs/resources-and-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_resources" "foo" {
+  name = "<your resource's name>"
+}
+```
+
```

Comment:
Add the following clarification paragraph:

`Resource names are not constrained to be unique within Twingate, so it is possible that this data source will return multiple list items.`

Reply:
updated docs